packages/cxl-lumo-styles/scss/_mixins.scss

```diff
@@ -43,3 +43,65 @@
     max-width: $max-width;
   }
 }
+
+/*
+* Typography Styles mixins
+*/
+
+@mixin light {
+  font-weight: 300;
+}
+
+@mixin regular {
+  font-weight: 400;
+}
+
+@mixin bold {
+  font-weight: 900;
+}
+
+@mixin italic {
+  font-style: italic;
+}
+
+@mixin label {
+  font-size: var(--lumo-font-size-s);
+  line-height: var(--cxl-lumo-line-height-m); // closest
+}
+
+@mixin title {
+  @extend bold;
+  font-size: var(--lumo-font-size-xxxl);
+  line-height: var(--cxl-lumo-line-height-xs);
+}
+
+@mixin subtitle {
+  @extend light;
+  font-size: var(--lumo-font-size-xxl);
+  line-height: var(--cxl-lumo-line-height-xs); // same as 35px, closest value to target (36px)
+}
+
+@mixin heading-2 {
+  @extend bold;
+  font-size: calc(2 * var(--lumo-font-size-xxs));
+  line-height: var(--cxl-lumo-line-height-s); // closest
+}
+
+@mixin heading-4 {
+  @extend bold;
+  font-size: var(--lumo-font-size-l);
+  line-height: var(--cxl-lumo-line-height-s); // closest
+}
+
+@mixin component-text {
+  @extend regular;
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--cxl-lumo-line-height-m);
+}
+
+@mixin body-text {
```

Comment:
These look similar and it's no clear for me where should we use each of these. Please add comments.